sdl2-vga-terminal/examples/sdl2-vga-terminal-load-so.c

```diff
@@ -10,36 +10,46 @@
 typedef void VGA_Terminal;
 
 #ifdef WIN32
+
 #include <Windows.h>
+
 typedef void(__cdecl* PROCWRITEXY)(VGA_Terminal*, uint8_t, uint8_t, char* str, uint8_t, uint8_t);
 typedef void(__cdecl* PROCDESTROY)(VGA_Terminal*);
 typedef void(__cdecl* PROCRENDER)(VGA_Terminal*);
 typedef VGA_Terminal* (__cdecl* PROCINIT)();
 #endif
 
 #ifdef linux
+
 #include <dlfcn.h>
+
 #endif
 
 #if defined(DEBUG) || !defined(NDBEBUG)
-#define VGA_SO_NAME "vga-terminal-libd"
+
+#define VGA_SO_NAME "vga-terminald"
+
 #else
-#define VGA_SO_NAME "vga-terminal-lib"
+
+#define VGA_SO_NAME "vga-terminal"
+
 #endif
 
 #ifdef WIN32
+
 void win32()
 {
-	const char* soname = VGA_SO_NAME ".dll";
+	const char* soname = TEXT(VGA_SO_NAME ".dll");
 	HINSTANCE hinstLib;
 	PROCINIT ProcAddInit = NULL;
 	PROCDESTROY ProcAddDestroy = NULL;
 	PROCWRITEXY ProcAddWriteXY = NULL;
 	PROCRENDER ProcAddRender = NULL;
 	BOOL fFreeResult, fRunTimeLinkSuccess = FALSE;
+	
 	printf("loading library: %s\n", soname);
 
-	hinstLib = LoadLibrary(TEXT("vga-terminal-libd.dll"));
+	hinstLib = LoadLibrary(soname);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Ensure that the full path to the library is specified, or current directory may be used (CWE-829, CWE-20).  Use registry entry or GetWindowsDirectory to find library path, if you aren't already. ](https://app.codacy.com/manual/Raffaello/sdl2-vga-terminal/pullRequest?prid=5382501)